operating/visualizing-external-entities.adoc

```diff
@@ -0,0 +1,239 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="external-ips"]
+= Visualizing External Entities
+include::modules/common-attributes.adoc[]
+:context: visualizing-external-entities
+
+toc::[]
+
+[role="_abstract"]
+
+{product-title-short} 4.6 introduces the External-IPs feature in Technology Preview, bringing more visibility about the external entities communicating with your cluster fleet.
+
+The user will be able to retrieve this information via API as well as via network graph.
+
+== Prerequisites
+External-IPs is an opt-in functionality, which is disabled by default in all secured clusters and central. In order to enable external IPs some steps need to be performed in the secured cluster you are interested in as well as central.
+
+In a nutshell:
+[cols="1,4,1,1", options="header"]
+|===
+|
+|Prerequisites
+|External IP API
+|External IP in Network Graph
+
+|Secured cluster
+|Enable external IPs collection per secured cluster runtime configuration
+|Yes
+|Yes
+
+.2+|Central services
+|Enable external IPs storage in Central via env variable: ROX_EXTERNAL_IPS
+|Yes
+|Yes
+
+|Enable external IPs in Network Grah via env_variable ROX_NETWORK_GRAPH_EXTERNAL_IPS
+|No
+|Yes
+|===
+
+
+* Enable external IPs collection per secured cluster runtime configuration: Each cluster can be individually configured to turn on collection of external IPs. To enable this feature on a given cluster, use “Collector Runtime configuration” (which is available from 4.6 on Tech. Preview) 
+
+=== Configure secured cluster:
+Refer to the <<Runtime Configuration>>
+
+runtime_config.yaml -> networking -> externalIps -> enable : true
+
+Note: It is supported to have clusters with the functionality enabled and others disabled at the same time. In this case, IP information will be available only for enabled clusters.
+
+=== Configure central: 
+
+* Enable external IPs in Central:  This is done using two environmental variables which are disabled by default: 
+  +
+- ROX_EXTERNAL_IPS enables storing external IP information in Central
+
+With operator:
+
+	Insert the following customization in your Central CRD instance:
```

Comment:
The spaces here are making this be set off into a box - delete the spaces in front of "Insert".

I'm actually not sure what you are trying to do here, but here is a pointer to how to format a series of steps for a procedure:

https://raw.githubusercontent.com/redhat-documentation/modular-docs/master/modular-docs-manual/files/TEMPLATE_PROCEDURE_doing-one-procedure.adoc

Reply:
So you would have something like:

```
. Enable external IPs by configuring the following environmental variables:
* ROX_EXTERNAL_IPS: Enables storing external IP information in Central. Depending on your installation method of Operator or Helm chart, do one of the following tasks:
** Insert the following into your CRD:
(example)
** Update your installation with an overlay as shown in the following example:
(example)
* ROX_NETWORK_GRAPH_EXTERNAL_IPS: Enables visualizing IPs in the network graph. Depending on your installation method of Operator or Helm chart, do one of the following tasks:
** Insert the following into your CRD:
(example)
** Update your installation with an overlay as shown in the following example:
(example)
```

Reply:
This might be helpful too:

https://docs.asciidoctor.org/asciidoc/latest/lists/unordered/

https://docs.asciidoctor.org/asciidoc/latest/lists/ordered/

Reply:
Thanks. I have removed the spaces before insert.